### ISSUE TITLE
Port to Visual Studio Code

### ISSUE THREAD

**Issue by edwinramirez** [beekman/Rainbow-Selection-Flux#16]
As the title says, could you bless VSC with arcane powers? Please????

**Comment by mhulse** [comment]
Good suggestion!

Looks like there's a Mac version:

https://code.visualstudio.com/download

Seems like it should be possible. I've never used VSC, but I assume it has a plugin system for 3rd part authors?

**Comment by edwinramirez** [comment]
Yes, you can build extensions and themes for VSC: https://code.visualstudio.com/docs/extensions/overview

I know that Atom is a lot more open to customization than VSC but not sure to what extent.
But I've used a couple of extensions that modify text and empty spaces with colors so I think that porting magical-selection is completely possible without it having to be an ordeal. 
Like this one that colorizes tab indentations: https://marketplace.visualstudio.com/items?itemName=oderwat.indent-rainbow
And this one that colorizes matching brackets: https://marketplace.visualstudio.com/items?itemName=2gua.rainbow-brackets

**Comment by mhulse** [comment]
Cool! Thanks for linkages. I'll check it out. I've been meaning to play with VSC anyway, so this could be a good intro. 👍

**Comment by mhulse** [comment]
Kinda looking at this now. 

Some links:

- https://github.com/Microsoft/vscode/issues/1636#issuecomment-294015842
- https://code.visualstudio.com/docs/extensions/overview
